src/ensembl/src/shared/types/thoas/product.ts

```diff
@@ -42,6 +42,10 @@ export enum ProductType {
   PROTEIN = 'Protein'
 }
 
+export type Sequence = {
+  checksum: string;
+};
```

Comment:
I think `Sequence` deserves its own file. There's nothing in it that is specific to Product.

Reply:
Done. I wasn't too sure if it merits its own file since it will just be 3 lines.